api/migrations/versions/0244_add_sms_sender_archived_state.py

```diff
@@ -0,0 +1,22 @@
+"""
+
+Revision ID: 0244
```

Comment:
My PR #393 also adds a migration 244 😛

Reply:
@jonalport Thanks for letting me know. I will change it to 245.